consensus/src/counters.rs

```diff
@@ -91,6 +91,10 @@ pub static TOTAL_VOTING_POWER: Lazy<Gauge> = Lazy::new(|| {
     .unwrap()
 });
 
+// Voting power of the validator
+pub static VOTING_POWER: Lazy<Gauge> =
+    Lazy::new(|| register_gauge!("aptos_voting_power", "Voting power of the validator").unwrap());
```

Comment:
let's name it aptos_validator_voting_power (in contrast to aptos_total_voting_power)?